src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,29 @@
+/*Class created by Priyanka Agarwal as per code written
+on codeU directions. Only added package and import statements
+ */
+
+package codeu.chat.common;
+import codeu.chat.util.Uuid;
+import java.io.IOException;
+
+
+public final class ServerInfo {
+    private final static String SERVER_VERSION = "1.0.0";
+
+    public Uuid version;
+    public ServerInfo() {
```

Comment:
I added a test class to ensure the versions were correct. Please let me know if I can add more to it